server/middleware/auth.py

```
@@ -1,7 +1,11 @@
-from fastapi import Header, HTTPException, status
-from services.auth_service import auth_service
+from fastapi import Header, HTTPException, status, Depends
+from dependency.dependencies import get_auth_service
+from services.auth_service import AuthService
 
-async def get_current_user(authorization: str = Header(None)) -> str:
+async def get_current_user(
+    authorization: str = Header(None), 
+    auth_service: AuthService = Depends(get_auth_service)
```

Ruff (0.14.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)